docker-compose.services.yml

```
@@ -8,6 +8,16 @@ services:
     restart: always
     ports:
       - '6379:6379'
+
+  apollo-server:
+    container_name: personnel-api
+    build:
+      context: ../ussf-personnel-api/
```

Suggestion: Having it this way means everyone, including our designers will need to have the repo checked out. I wonder if it would be better to pull from github by default and leave a comment on how to use the local copy for us devs who might be working on both repos at the same time. Thoughts? The below is an example similar to how the `cms` is setup in `docker-compose.cms.yml`, but a deviation from how we expect the cms to be running locally.

```suggestion
      # Uncomment the below line to work with local check out
      # context: ../ussf-personnel-api/
      context: https://github.com/USSF-ORBIT/ussf-personnel-api.git#main
```